addons/survey/static/src/views/widgets/survey_question_trigger/survey_question_trigger.js

```diff
@@ -0,0 +1,113 @@
+/** @odoo-module **/
+
+import { _lt } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { sprintf } from '@web/core/utils/strings';
+import { standardWidgetProps} from "@web/views/widgets/standard_widget_props";
+
+const { Component, useEffect, useRef, useState } = owl;
+
+export class SurveyQuestionTriggerWidget extends Component {
+    setup() {
+        super.setup();
+        this.button = useRef('survey_question_trigger');
+        this.state = useState({
+            triggeringQuestionTitle: this.props.record.data.triggering_question_id,
+            surveyQuestionTriggerTooltip: ""
+        });
+        useEffect(() => {
+            if (this.button && this.button.el) {
+                this.state.triggeringQuestionTitle = this.props.record.data.triggering_question_id[1];
+                const triggerError = this.surveyQuestionTriggerError;
+                if (triggerError === "MISSING_TRIGGER_ERROR") { //extra-precaution, see `surveyQuestionTriggerError`
+                    this.state.surveyQuestionTriggerTooltip = sprintf(
+                        _lt('⚠️ The trigger question configured ("%s") is missing. This trigger will be automatically removed.'),
```

Comment:
Mmh, would put the warning icon outside of translated message, not sure if all our translations tool will handle it properly and it does not need to be translated anyway.